docker-compose.production.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.2"
+services:
+ db:
+   image: mysql:5.7
+   env_file:
+     - '.env'
+   ports:
+     - "3306:3306"
+   command: ['mysqld', '--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+   healthcheck:
+     test: ["CMD", "mysqladmin", "ping", "-h", "localhost","-plinks"]
+     timeout: 20s
+     interval: 10s
+     retries: 10
+ migrate:
+   image: wikipedialibrary/externallinks:latest
+   entrypoint: ["python", "/app/manage.py", "migrate"]
+   restart: on-failure
+   env_file:
+     - '.env'
+   depends_on:
+     - 'db'
+   volumes:
+     - ./:/app
+ externallinks_app:
+   image: wikipedialibrary/externallinks:latest
+   build: .
+   env_file:
+     - '.env'
+   depends_on:
+     - 'migrate'
+   ports:
+     - "8000:8000"
+   volumes:
+     - ./:/app
+ nginx:
+   image: nginx
+   volumes:
+     - ./nginx.conf:/etc/nginx/conf.d/default.conf:ro
+     - ./static:/app/static
+   ports:
+     - "80:80"
+   depends_on:
+     - 'externallinks_app'
+ externallinks_eventstream:
+   image: wikipedialibrary/externallinks:latest
+   build:
+    context: .
+    dockerfile: 'Dockerfile-eventstream'
+   depends_on:
+     - 'db'
+   env_file:
+     - '.env'
+   volumes:
+     - ./:/app
+ cache:
+   image: memcached
+   ports:
+    - "11211:11211"
+   entrypoint:
+     - memcached
+   depends_on:
+     - 'externallinks_app'
```

Comment:
this looks identical to the service definition in the main docker-compose file. We only need to override changes.